modules/Hoot/ui/conflicts/merge.js

```diff
@@ -23,13 +25,25 @@ export default class Merge {
      * @param instance - conflicts class
      */
     constructor( instance ) {
+        this.instance = instance;
         this.data = instance.data;
 
         this.mergeArrow = {
             from: null,
-            to: null,
-            reverseFeatures: null
+            to: null
         };
+
+        this.isPoiPoly = false;
+
+        let that = this;
+        Hoot.context.history()
+            .on('undone.review_merge', function() {
+                that.checkMergeButton.call(that);
+            });
+        Hoot.context.history()
+            .on('redone.review_merge', function() {
+                that.checkMergeButton.call(that);
+            });
```

Comment:
@jackgrossman18 listen for undo/redo and call the same function that is called when a new review is loaded.